apps/app/messages/en.json

```diff
@@ -175,26 +183,31 @@
     "enterAnAmount": "Enter an amount",
     "approvalButton": "Approve {symbol}",
     "approvalTx": "{symbol} Approval",
+    "reviewDelegation": "Review Delegation",
     "reviewDeposit": "Review Deposit",
     "reviewWithdrawal": "Review Withdrawal",
     "confirmDeposit": "Confirm Deposit",
     "confirmWithdrawal": "Confirm Withdrawal",
     "depositTx": "{symbol} Deposit",
     "withdrawTx": "{symbol} Withdrawal",
+    "delegateTx": "{symbol} Delegate",

```

Comment:
Similarly, since this will be in the past tense on "Recent Transactions", it should be "Delegation".